apps/extension/src/background/keyring/messages.ts

```diff
@@ -350,3 +351,30 @@ export class DeleteAccountMsg extends Message<
     return DeleteAccountMsg.type();
   }
 }
+
+export class QueryAccountDetailsMsg extends Message<
```

Comment:
Sorry if I'm missing something, but are we using QueryAccountDetailsMsg anywhere? I've noticed this message type hasn't been added to the handler function

Reply:
You're right! This was copied from my Ledger update PR, I can remove it here

Reply:
Fixed!